assets/js/script.js

```diff
@@ -4,126 +4,175 @@
  * TODO: Add support for subtraction
  */
 const OPERATIONS = {
-    "addition": {"symbol": "+", "createQuestion": createAdditionQuestion},
-    "multiplication": {"symbol": "x", "createQuestion": createMultiplicationQuestion},
-    "division": {"symbol": "÷", "createQuestion": createDivisionQuestion},
+  addition: { symbol: "+", createQuestion: createAdditionQuestion },
+  multiplication: { symbol: "x", createQuestion: createMultiplicationQuestion },
+  division: { symbol: "÷", createQuestion: createDivisionQuestion },
 };
 
 /**
  * Generate a pseudorandom integer between two values, inclusive
  */
- function randomInteger(min, max) {
-    return Math.floor(Math.random() * (max - min + 1)) + min;
+function randomInteger(min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+/**
+ * The four basic arithmetic operations for easy unit testing:
+ * addition, subtraction, multiplication and division.
+ * To be called inside of each "create[operation name]Question" function
+ */
+function add(num1, num2) {
+  return num1 + num2;
+}
+
+function subtract(num1, num2) {
+  return num1 - num2;
+}
+
+function multiply(num1, num2) {
+  return num1 * num2;
+}
+
+function divide(num1, num2) {
+  return num2 === 0 ? undefined : num1 / num2;
```

Comment:
I see you've implemented them all correctly. Would this maybe be the opportunity to add some bugs?
Ideas:
- We can do the division by zero issue.
- We can reverse the order of arguments for division or subtraction (if you want to keep the latter)
- I recall this was intended to only work with integers, so maybe we can add some bugginess to the rounding